docs/source/dev_setup.md

```diff
@@ -21,55 +31,65 @@ git checkout dev
 
 ### Configuration
 
-All configurations values are stored in environment variables.
+All configurations values are stored in environment variables. Please refer to the documentation on [Configuration](configuration.html) for further details.
+
 
-Please refer to the documentation on [Configuration](configuration.md) for further details.
+### Directories
+
+MapSwipe Workers needs access to a data directory for logs and data for data for the API:
 
+To create this directories run:
+```
+mkdir --parents ~/.local/share/mapswipe_workers
+```
 
-### Keys
+> Note: XDG Base Directory Specification is respected
 
-To gain access to Firebase MapSwipe workers needs a Service Account Key (`serviceAccountKey.json`). Create on in Firebase and store it where ever you want. Make sure to store the path to the key in the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
 
+### Service Account Key
 
-### Directories
+The MapSwipe Workers requires a Service Account Key (`serviceAccountKey.json`) to access Firebase database. Request yours from the MapSwipe working group.
 
-MapSwipe Workers needs access to a directory for data and logs.
-To create the directory run `mkdir --parents ~/.local/share/mapswipe_workers`.
+The path the Service Account Key is defined in the `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
 
-> Note: XDG Base Directory Specification is respected
+You could also set up your own Firebase instance. However, this is not recommended. If you still want to do it, get your Service Account Key from Firebase from [Google Cloud Service Accounts](https://console.cloud.google.com/iam-admin/serviceaccounts).
 
 
-### Install MapSwipe Workers Python Package
+### Postgres
 
-1. Create a Python virtual environment with `system-site-packages` option enabled to get access to GDAL/OGR Python packages
-2. Activate the virtual environment.
-3. Install MapSwipe Workers using pip.
-4. Run it.
+Setup a local Postgres instance for MapSwipe Workers using the Dockerfile provided for development purposes (`postgres/Dockerfile-dev`). Make sure that the specified port is not in use already. If so, adjust the port (also in the `.env` file).
 
 ```bash
-python -m venv --system-site-packages venv
-source venv/bin/activate
-pip install --editable .
-mapswipe_workers --help
+docker build -t mapswipe_postgres -f ./postgres/Dockerfile-dev ./postgres
```

Comment:
any reasons why you are not using this instead?
```
docker-compose build postgres
docker-compose up -d postgres
```

Reply:
Because for the dev setup there exists a different postgres Dockerfile (`/postgres/Dockerfile-dev `) without the backup setup. This Dockerfile is not included in the docker-compose yet. We could add it, but its only for development purposes.